resources/images/s3/filestorage.go

```diff
@@ -43,14 +46,15 @@ const (
 // Data layer for file storage.
 // Implements model.FileStorage interface
 type SimpleStorageService struct {
-	client *s3.S3
-	bucket string
+	client       *s3.S3
+	bucket       string
+	setTenantTag bool
```

Comment:
on second thought, maybe this could be renamed to `haveMultitenant` or `withMultitenant`. it'll surely come in handy for other MT-related logic (after my requests it'll already be used both when uploading an artifact, and getting one).

Reply:
PS. along with the associated setting in config